aspnetcore/fundamentals/error-handling.md

```diff
@@ -113,70 +102,63 @@ Here's an example of text displayed by the default handlers:
 Status Code: 404; Not Found
 ```
 
-The middleware supports several extension methods that allow you to customize its behavior.
+## UseStatusCodePages with format string
 
-### UseStatusCodePages with lambda
+An overload of <xref:Microsoft.AspNetCore.Builder.StatusCodePagesExtensions.UseStatusCodePages*> takes a content type and format string, which you can use to customize the content type and response text:
```

Comment:
An overload of <xref:Microsoft.AspNetCore.Builder.StatusCodePagesExtensions.UseStatusCodePages*> takes a content type and format string, can be used to customize the content type and response text: